include/rocksdb/options.h

```diff
@@ -844,6 +844,15 @@ struct DBOptions {
   //
   // Dynamically changeable through SetDBOptions() API.
   bool avoid_flush_during_shutdown = false;
+
+  // By default CompactionIterator will try to zero out sequence numbers
```

Comment:
CompactionIterator is an not part of the public API, I think it's better to just say `Compaction`